linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_password_pam_unix_remember/oval/shared.xml

```diff
@@ -1,8 +1,14 @@
-{{% if product in [ "sle12", "sle15" ] %}}
+{{% if product in [ "sle12", "sle15" ] or product in [ "ubuntu1804", "ubuntu2004"] %}}
```

Comment:
After closing the original PR the comment feedbacks aren't working. Here's feedback left on this OVAL from @dodys:
> Is there any reason to drop the specific oval?

No, I do not think there's a reason for dropping the specific OVAL. I think originally what I was intending to do was reduce the need to have two separate OVALs. Though, this would make the current `shared.xml` implementation complex. Would you suggest that this be reverted?